Tella/Data/Networking/Models/UwaziLoginResult.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UwaziLoginResult.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 5/23/23.
+//  Copyright © 2023 INTERNEWS. All rights reserved.
+//
+
+import Foundation
+struct UwaziLoginResult: Codable {
```

Comment:
You can use `BoolResponse` instead to create a new one, so you can move it to a new class